packages/shared/src/components/modals/PostToSquadModal.tsx

```diff
@@ -91,8 +107,12 @@ function PostToSquadModal({
       steps={post ? undefined : modalSteps}
     >
       <Modal.Header title="Share post" kind={ModalHeaderKind.Tertiary} />
-      {post ? (
-        <SquadComment form={form} onSubmit={onSubmit} isLoading={isLoading} />
+      {shouldSkipHistory ? (
```

Comment:
This seems to cause a flicker locally atleast.

Reply:
Should be okay now. There were two areas triggering the show of the modal.